security-groups.tf

```diff
@@ -35,6 +35,24 @@ resource "aws_security_group" "nat" {
   }
 }
 
+/* Security group for the Gandalf server */
+resource "aws_security_group" "gandalf" {
+  name = "tsuru-gandalf"
+  description = "Security group for Gandalf instance that allows SSH access from internet"
+  vpc_id = "${aws_vpc.default.id}"
+
+  ingress {
+    from_port = 22
+    to_port   = 22
+    protocol  = "tcp"
+    cidr_blocks = ["80.194.77.0/24"]
```

Comment:
Is there an official list of GDS IPs we should be using?
If yes, I can go ahead and create the PR. I remember we were using /32 in the beginning but we were getting occasional issues connecting. I believe even we're all using the same `CDN02_test` wi-fi network there's a handful of /32 IPs that come up.